npm/tailcall/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@tailcallhq/tailcall",
+  "version": "0.13.0",
+  "type": "module",
+  "homepage": "https://tailcall.run",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tailcallhq/tailcall.git"
+  },
+  "author": "TailcallHQ",
+  "license": "MIT",
+  "bugs": "https://github.com/tailcallhq/tailcall/issues",
+  "description": "A high-performance GraphQL proxy, optimized for cloud-native environments, covering edge, middle, and service layers.",
+  "files": [
+    "scripts/postinstall.js"
```

Comment:
do we need this?

Reply:
has it been resolved? I still see that entry but no such file in the pr